shell/components/AppModal.vue

```diff
@@ -136,11 +143,16 @@ export default defineComponent({
   },
   mounted() {
     document.addEventListener('keydown', this.handleEscapeKey);
+    document.addEventListener('focusin', this.focusChanged);
```

Comment:
I think we want to reconsider adding event listeners at the document level.

Reply:
removed. fixed